utils.js

```diff
@@ -84,16 +86,17 @@ function unload(callback, container) {
       callback();
     }
     catch(ex) {
-      console.error(ex);
+      // console.error(ex);
```

Comment:
(This is the one situation where I wouldn't just delete commented-out code, because I think we might want to re-add it for debugging.)